.github/workflows/pr-generator.yml

```
@@ -0,0 +1,97 @@
+name: "PR Generator"
+on:
+  workflow_dispatch:
+    inputs:
+      go-algorand-version:
```

```suggestion
      go_algorand_version:
```

should this use this formatting instead?